src/components/HomeFooter.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {StyleSheet, Text, View} from 'react-native';
 import {useTranslation} from 'react-i18next';
 import '../utils/i18n';
-const HomeScreenFooter = (): React.JSX.Element => {
+const HomeFooter = (): React.JSX.Element => {
```

Comment:
You can make this footer reusable by passing the text as a prop and naming it `Footer` and use it anywhere